packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/angular/basic-sign-in.md

```diff
@@ -0,0 +1,116 @@
+This section helps you review the simple password-only sign-in use case from the sample app. This use case is outlined in the following sequence diagram with your single-page app as the client:
+
+![Sequence diagram that displays the back and forth between the resource owner, sdk, authorization server, and resource server for a basic SPA password sign-in flow.](/img/oie-embedded-sdk/password-only-spa-authjs-flow.svg "Auth JS + SPA password-only sign-in flow")
+
+### Review the Okta configuration settings
+
+Review the `src/app/okta-config.ts` file that references the required [configuration settings](#configuration-settings) to initialize your Okta Auth JS instance. The `okta-config.ts` file references the values from your Okta app integration.
+
+```TypeScript
+import { OktaAuth } from '@okta/okta-auth-js';
+import { FactoryProvider, InjectionToken } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+import { Router } from '@angular/router';
+
+const oidcConfig = {
+  clientId: '${clientId}',
+  issuer: 'https://${yourOktaDomain}/oauth2/default',
+  scopes: ['openid', 'profile', 'email'],
+  pkce: true
+};
+```
+
+### Instantiate the Okta Auth JS client
+
+Review the same Angular `src/app/okta-config.ts` file that imports required libraries and instantiates the Okta Auth JS client with values from the `oidcConfig` variable.
+
+```TypeScript
+import { OktaAuth } from '@okta/okta-auth-js';
+import { FactoryProvider, InjectionToken } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+import { Router } from '@angular/router';
+
+const oidcConfig = {
+  clientId: '${clientId}',
+  issuer: 'https://${yourOktaDomain}/oauth2/default',
+  scopes: ['openid', 'profile', 'email'],
+  pkce: true
+};
+
+export const OKTA_AUTH = new InjectionToken('okta-auth');
+
+const oktaAuthFactory = (router: Router, doc: any) => {
+  const params = router.parseUrl(doc.location.search);
+  const state = params.queryParamMap.get('state') ?? undefined;
+  const recoveryToken = params.queryParamMap.get('recoveryToken') ?? undefined;
+  const redirectUri = `${doc.location.origin}/login/callback`;
+  return new OktaAuth({...oidcConfig, redirectUri, state, recoveryToken})
+};
+
+export const OKTA_PROVIDER: FactoryProvider = {
+  provide: OKTA_AUTH,
+  useFactory: oktaAuthFactory,
+  deps: [Router, DOCUMENT]
+}
+```
+
+To start the Okta Auth JS service, see this code snippet from the `okta-auth-service.ts` file:
+
+```TypeScript
+import { OKTA_AUTH } from './okta-config';
+
+constructor(@Inject(OKTA_AUTH) private oktaAuth: OktaAuth) {
+  this.oktaAuth.start();
+}
+```
+
+### Handle the password authentication
+
+Review the `okta-auth.service.ts` file for details on handling a successful password authentication by receiving the `SUCCESS` status, which also returns tokens. For example, for the simple authentication case:
```

Comment:
This feels a little misleading since the code isn't actually handling `SUCCESS` (it's treating it the same way as `CANCELED`). This is because while the success response includes the token, we're actually using the callback passed into the `oktaAuth.authStateManager.subscribe(callbackFn)` to handle the token. 

Do you want me to provide a code snippet where we're pulling the token out of the transaction or a code snippet to show how the callback for token management works?

Reply:
Would this be the code snippet? I think adding content on this would be great. Or if you want to provide something generic.

```
 constructor(@Inject(OKTA_AUTH) private oktaAuth: OktaAuth, private router: Router, @Inject(DOCUMENT) private doc: Document) {
    this.authStateHandler = this.authStateHandler.bind(this);
    this.oktaAuth.authStateManager.subscribe(this.authStateHandler);
    this.oktaAuth.start();

    this.log$ = this.appAuthStateSub$.asObservable().pipe(
      filter(state => !!state.transaction),
      distinctUntilKeyChanged('transaction'),
      map(state => state.transaction as IdxTransaction),
      takeUntil(this.destroySub$)
    ).subscribe(transaction => console.log(transaction));

    this.tokenHandler$ = this.appAuthStateSub$.asObservable().pipe(
      filter(state => !!state.transaction && !!state.transaction.tokens),
      map(state => state.transaction?.tokens as Tokens),
      takeUntil(this.destroySub$)
    ).subscribe(tokens => this.oktaAuth.tokenManager.setTokens(tokens))
  }
```

Reply:
Sorry, I misspoke. It's not the authStateManager callbackFn in this case. I am pulling it from the transaction and manually applying it to the authStateManager, but I'm doing so in a more decoupled way. So to make the example easier to follow, I think you can keep the same text and update the `authenticateFlow()` snippet to

```ts
public authenticateFlow(): Observable<NextStep | undefined> {
  return defer(() => this.oktaAuth.idx.authenticate().pipe(
    tap( transaction => {
      if (transaction.status === IdxStatus.SUCCESS && !!transaction.tokens) {
        this.oktaAuth.tokenManager.setToken(transaction.tokens);
      }
    }),
    map( transaction => {
      const status = transaction.status;

      if (status === IdxStatus.SUCCESS || status === IdxStatus.CANCELED) {
        return undefined;
      }

      if (status === Idx.FAILURE) throw 'Idx error';
      return transaction.nextStep;
    }
  );
}
```

Reply:
Thanks. Added this snippet.